.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Run Benchmarks
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # Runs weekly on Sunday at midnight
```

Comment:
If we chart this, we will probably want it more often than this.

Reply:
Let's try daily?